Core/include/Acts/TrackFinding/CombinatorialKalmanFilter.hpp

```diff
@@ -881,8 +996,7 @@ class CombinatorialKalmanFilter {
           result.activeTips.erase(result.activeTips.end() - 1);
         }
       }
-
-      return Result<void>::success();
+      return std::make_tuple(nBranchesOnSurface, isOutlier);
```

Comment:
```suggestion
      return {nBranchesOnSurface, isOutlier};
```
I think `std::make_tuple` is an C++14 relict

Reply:
it is not deprecated as far as I see, and it is more explicit about what it does than {}, but I can change to the more compact version.

Reply:
g++13 does not do the conversion automatically. I could change it to:
` Result<std::tuple<unsigned int, bool> >::success({nBranchesOnSurface, isOutlier});`
but that is not really an improvement, or is it ?